### DISCUSSION TITLE
Portuguese Language

### DISCUSSION THREAD

**Discussion by danivraposo** [monarc-project/MonarcAppFO#538] [General]
I am reaching out to offer my services for translating your materials into Portuguese. I believe that a Portuguese translation would be highly beneficial, particularly for users of cncs.gov.pt.

To proceed, I would need to know what files would matter to thistranslation.

**Comment by ruslanbaidan** [comment]
Duplication of https://github.com/monarc-project/MonarcAppFO/discussions/536
